src/com/lftechnology/java/training/niraj/ems/services/implementations/RouteServices.java

```diff
@@ -0,0 +1,111 @@
+package com.lftechnology.java.training.niraj.ems.services.implementations;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.niraj.ems.controllers.EmployeeController;
+import com.lftechnology.java.training.niraj.ems.controllers.LoginController;
+import com.lftechnology.java.training.niraj.ems.domains.Employee;
+import com.lftechnology.java.training.niraj.ems.enums.Roles;
+import com.lftechnology.java.training.niraj.ems.utils.Constants;
+import com.lftechnology.java.training.niraj.ems.utils.UserNumInputImpl;
+import com.lftechnology.java.training.niraj.ems.utils.Utils;
+
+public class RouteServices {
+    private static final Logger LOGGER = Logger.getLogger(RouteServices.class.getName());
+    private static UserNumInputImpl userInput = new UserNumInputImpl();
+
+    private RouteServices() {
+
+    }
+
+    public static void routeLandingPage() {
+        try (Scanner scanner = new Scanner(System.in)) {
+            Utils.clearConsole();
+            LoginController loginController = new LoginController();
+            LOGGER.info(Constants.LANDING_MENU);
+            int userChoice = RouteServices.userInput.getInput(scanner, 1, 2);
+            switch (userChoice) {
+            case 1:
+                loginController.login();
+                break;
+            case 2:
+                loginController.register();
+                break;
+            default:
+                LOGGER.warning(Constants.INVALID_INPUT);
+                break;
+            }
+        } catch (InputMismatchException se) {
+            LOGGER.warning(Constants.INVALID_INPUT);
+        }
+
+    }
+
+    public static void routeAfterLogin(Employee employee) {
+        if (employee != null) {
+            Utils.clearConsole();
+            try (Scanner scanner = new Scanner(System.in)) {
+                String role = employee.getRole();
+                if (role.equals(Roles.ADMIN.getRole())) {
+                    routeAfterLoginAdmin(scanner, employee);
+                } else {
+                    routeAfterLoginUser(scanner, employee);
+                }
+
+            } catch (InputMismatchException se) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)